Cloud index screenshots update and remove cloud quickstart (#10692)
PrefectHQ/prefect: docs/cloud/cloud-quickstart.md

```bash
prefect cloud login
```

The `prefect cloud login` command, used on its own, provides an interactive login experience.
Using this command, you may log in with either an API key or through a browser.

```output

? How would you like to authenticate? [Use arrows to move; enter to select]
> Log in with a webb browser                                                
  Paste an API key                                                         
Opening browser...
Waiting for response...
Authenticated with Prefect Cloud! Using workspace 'jeffdc/prod'.

```

If you choose to log in via the browser, Prefect opens a new tab in your default browser and enables you to log in and authenticate the session.

## Run a flow with Prefect Cloud

You're all set to run a flow locally, orchestrated with Prefect Cloud.

In your local environment, where you configured the previous steps, create a file named `quickstart_flow.py` with the following contents:

```python
from prefect import flow

@flow(log_prints=True)
def quickstart_flow():
    print("Local quickstart flow is running!")

if __name__ == "__main__":
    quickstart_flow()
```

Now run `quickstart_flow.py`.

> You'll see log messages similar to the following in the terminal, indicating that the flow is running correctly.

Reviewer: ```suggestion
You'll see log messages like this in your terminal, indicating that the flow is running correctly:
```

17:18:09.864 | INFO    | Flow run 'fragrant-quetzal' - View at https://app.prefect.cloud/account/my_workspace_id/workspace/my_flow_id/flow-runs/flow-run/my_flow_run_id
17:18:10.010 | INFO    | Flow run 'fragrant-quetzal' - Local quickstart flow is running!
17:18:10.144 | INFO    | Flow run 'fragrant-quetzal' - Finished in state Completed()
```

Go to the **Flow Runs** pages in your workspace in Prefect Cloud.
You'll see the flow run results right there in Prefect Cloud!

![Viewing flow run results in the Prefect Cloud UI Flow Runs dashboard.](/img/ui/cloud-flow-run.png)

Prefect Cloud automatically tracks any flow runs in a local execution environment logged into Prefect Cloud.

Select the name of the flow run to see details about this run.

![Viewing local flow run details in the Prefect Cloud UI.](/img/ui/cloud-flow-run-details.png)

Congratulations! You successfully ran a local flow and, because you're logged into Prefect Cloud, the local flow run results were captured by Prefect Cloud.

## Next steps

If you're new to Prefect, learn more about writing and running flows in the [Prefect Flows First Steps](/tutorial/flows/) tutorial.

> If you're already familiar with Prefect flows and want to try creating deployments and kicking off flow runs with Prefect Cloud, check out the [Deployments](/tutorial/deployments/) tutorial.

Reviewer: ```suggestion
If you're already familiar with flows, try creating a deployment and triggering flow runs with Prefect Cloud by following the [Deployments](/tutorial/deployments/) tutorial.
```
